src/api.ts

```diff
@@ -75,7 +117,8 @@ export const registerServiceFunction = (
     fnName: string,
     handler: (args: any[], tetraplets: SecurityTetraplet[][]) => object,
 ) => {
-    (client as FluenceClientImpl).registerCallback(serviceId, fnName, handler);
+    const unuse = (client as ClientImpl).handler.on(serviceId, fnName, handler);
```

Comment:
why is it called "unuse"? I couldn't figure out what this variable means by the name

Reply:
That's basically a hack to glue old api to the new one. I've added some comments